src/Parser/XbilParser.js

```diff
@@ -64,8 +64,16 @@ export function computeMinMaxElevation(texture, pitch, options) {
                 }
             }
         }
-        if (options.zmin > min) { min = options.zmin; }
-        if (options.zmax < max) { max = options.zmax; }
+        // Clamp values to zmin and zmax values configured in ElevationLayer
+        if (options.zmin != null) {
+            if (min < options.zmin) { min = options.zmin; }
+            if (max < options.zmin) { max = options.zmin; }
+        }
+
+        if (options.zmax != null) {
+            if (min > options.zmax) { min = options.zmax; }
+            if (max > options.zmax) { max = options.zmax; }
+        }
```

Comment:
I thought about it but I found it harder to read and understand, wdyt?

Reply:
In my opinion I find it better as it suppreses the need to check for null and 6 ifs. But I also not reallypartial as it is my proposal ^^
It can also if needed, keep the initial value of min and max and use zmin/zmax for the following traitement (with a minor change)

Reply:
I've tested it and it does not work:
options.zmin = 0
options.zmax = undefined = max = -98509.849609375
max = -98509.849609375
Math.max(max, zmin) = 0
max = Math.min(zmax, Math.max(max, zmin)); = -98509.849609375 (instead of 0)

For it to work this way we may need to apply another Math.min and Math.max to zmax and zmin definition at the beginning. 

Shall we go with the good old ifs ? (easier to read and to debug in this case in my opinion)

Reply:
If it works, and you find it more readable, go for it.